i18n/zh_CN/LC_MESSAGES/Alltube.po

```diff
@@ -1,134 +1,234 @@
+msgid "Video password"
+msgstr "视频密码"
 
 #: templates/index.tpl:8
+# I don't think this needs to be a 100% match
 msgid "Copy here the URL of your video (YouTube, Dailymotion, etc.)"
-msgstr "在这里复制您的视频 （Youtube、 Dailymotion 等） 的 URL"
+msgstr "在此处粘贴视频网址"
 
-#: templates/index.tpl:23
+#: templates/index.tpl:25
 msgid "Audio only (MP3)"
-msgstr "仅限音频（mp3）"
+msgstr "仅音频（MP3）"
 
-#: templates/index.tpl:28
+#: templates/index.tpl:29
+# Still check that file for context
+msgid "From"
+msgstr "从"
+
+#: templates/index.tpl:32
+msgid "to"
+msgstr "到"
+
+#: templates/index.tpl:41
 msgid "See all supported websites"
-msgstr "请参阅支持的所有网站"
+msgstr "查看所有支持的网站"
 
-#: templates/index.tpl:30
+#: templates/index.tpl:43
 msgid "Drag this to your bookmarks bar:"
-msgstr "把这个拖到你的书签："
+msgstr "您可以把这个书签工具拖到您的书签栏中："
 
-#: templates/index.tpl:31
+#: templates/index.tpl:45
 msgid "Bookmarklet"
 msgstr "书签工具"
 
-#: templates/inc/header.tpl:4
-msgid "Switch language"
-msgstr "切换语言"
+#: classes/Controller/DownloadController.php:64
+#: classes/Controller/FrontController.php:166
+msgid "Wrong password"
+msgstr "密码错误"
 
-#: templates/error.tpl:5
-msgid "An error occurred"
-msgstr "出错了"
+#: classes/Controller/DownloadController.php:69
+msgid "Conversion of playlists is not supported."
+msgstr "不支持转换播放列表。"
+
+#: classes/Controller/DownloadController.php:76
+msgid "Conversion of M3U8 files is not supported."
+msgstr "不支持转换 M3U8 文件。"
+
+#: classes/Controller/DownloadController.php:82
+# ref. Chinese Wikipedia article about DASH
+msgid "Conversion of DASH segments is not supported."
+msgstr "不支持转换 DASH 片段。"
+
+#: classes/Controller/FrontController.php:65
+msgid ""
+"Easily download videos from YouTube, Dailymotion, Vimeo and other websites."
+msgstr ""
+"轻松从 YouTube、Dailymotion、Vimeo 等网站下载视频。"
+
+#: classes/Controller/FrontController.php:112
+# NOTE: DON'T translate AllTube Download
+msgid ""
+"List of all supported websites from which AllTube Download can extract video "
+"or audio files"
+msgstr ""
+"AllTube Download 能够提取视频"
+"或音频文件的的所有网站"
+
+#: classes/Controller/FrontController.php:138
+msgid "Password prompt"
+msgstr "密码提示"
+
+#: classes/Controller/FrontController.php:140
+msgid ""
+"You need a password in order to download this video with AllTube Download"
+msgstr ""
+"您需要密码才能使用 AllTube Download 下载此视频"
+
+#: classes/Controller/FrontController.php:174
+# Download page header?
+msgid "Video download"
+msgstr "下载视频"
+
+#: classes/Controller/FrontController.php:176
+msgid "Download video from @extractor"
+msgstr "从 @extractor 下载视频"
+
+#: classes/Controller/FrontController.php:182
+msgid "Download @title from @extractor"
+msgstr "从 @extractor 下载 @title"
+
+#: classes/Controller/FrontController.php:255
+msgid "Error"
+msgstr "错误"
+
+#: classes/Controller/FrontController.php:271
+msgid "Page not found"
+msgstr "找不到页面"
+
+#: classes/Controller/FrontController.php:282
+msgid "Method not allowed"
+msgstr "不允许此请求方法"
+
+#msgid "yes"
+#msgstr "是"
 
+#msgid "no"
+#msgstr "否"
```

Comment:
Is this supposed to be commented?

Reply:
Weird, i don't know why do I have that, i should probably remove them